shell/key-bindings.fish

```diff
@@ -84,4 +86,26 @@ function fzf_key_bindings
     bind -M insert \cr fzf-history-widget
     bind -M insert \ec fzf-cd-widget
   end
+
+  function __fzf_get_starting_dir
+    set -l dir (commandline -t)
+
+    # Default to current dir if no argument input
+    if [ -z "$dir" ]
+        set dir '.'
+    end
+
+    # The commandline token might be escaped, we need to unescape it.
+    set dir (eval "printf '%s' $dir")
```

Comment:
This will shell expand to absolute path, ie. $HOME<C-t>. If this command works without expansion, do you think there is value or good UX to leave unexpanded completion? I'm indifferent between finding `$HOME/.vimrc` or `/Users/ipwnponies/.vimrc`.